app/ui/src/app/connections/detail-page/info.component.ts

```diff
@@ -51,20 +53,33 @@ import { ConnectionConfigurationService } from '../common/configuration/configur
   `
   ]
 })
-export class ConnectionDetailInfoComponent {
+export class ConnectionDetailInfoComponent implements OnChanges {
+  readonly apiEndpoint: any;
   @Input() connection: Connection;
   @Output() updated = new EventEmitter<Connection>();
+  icon: String;
 
   constructor(
     private connectionService: ConnectionService,
-    private configurationService: ConnectionConfigurationService
-  ) {}
+    private configurationService: ConnectionConfigurationService,
+    private config: ConfigService
+  ) {
+    this.apiEndpoint = config.getSettings().apiEndpoint;
+  }
 
   onAttributeUpdated(attr: string, value) {
     this.connection[attr] = value;
     this.updated.emit(this.connection);
   }
 
+  ngOnChanges(changes: SimpleChanges) {
```

Comment:
This executes lots of times, and I suspect we don't need to evaluate this code that often.  This should probably go in `ngOnInit` where the `connection` object should also be available and set.